### DISCUSSION TITLE
Add Bluesky Social

### DISCUSSION THREAD

**Discussion by HenrickTheBull** [withastro/starlight#1853] [Feature Requests]
### What version of `starlight` are you using?

0.22.2

### What is your idea?

Adding support for Bluesky Social to the social config. 

It currently uses `https://bsky.app/profile/` as it's uri for profiles and has a glyph on FontAwesome version 6

Currently all profiles on Bsky are `subdomain.domain.tld` or `domain.tld` examples of this include my own project `https://bsky.app/profile/taurusshield.app` or the profile of the CEO of Bluesky PBC `https://bsky.app/profile/jay.bsky.team`

There is an alternative name for the network as a whole much like Activity Pub is called the FediVerse the ATProto network is being called the Atmosphere.

### Why is this feature necessary?

Honestly Bluesky is growing platform that will more than likely continue to grow as it becomes more decentralized and more people use it.

### Participation

- [ ] I am willing to submit a pull request for this feature.

**Comment by at-the-vr** [comment]
I see this easy to accomplish in Starlight. Will wait for maintainers thoughts on this one before PR'ing it up 😄

**Reply by delucis** [reply]
Oops, sorry @at-the-vr — already spun up a branch for this earlier today and just pushed it: https://github.com/withastro/starlight/pull/1871

Otherwise definitely would have appreciated your help 💜

**Reply by at-the-vr** [reply]
No worries at all 😄, solid work with the PR 💪

**Comment by at-the-vr** [comment]
And Bluesky has been introduced with [Starlight v0.22.4](https://github.com/withastro/starlight/releases/tag/%40astrojs%2Fstarlight%400.22.4) 🥳 Have a blast with this one 🔥

**Reply by HenrickTheBull** [reply]
Thanks so much.
